rocAL/rocAL/source/parameters/parameter_crop.cpp

```diff
@@ -76,7 +76,6 @@ void CropParam::create_array(std::shared_ptr<Graph> graph)
     vxAddArrayItems(croph_arr,batch_size, croph_arr_val.data(), sizeof(vx_uint32));
     vxAddArrayItems(x2_arr,batch_size, x2_arr_val.data(), sizeof(vx_uint32));
     vxAddArrayItems(y2_arr,batch_size, y2_arr_val.data(), sizeof(vx_uint32));
-    update_array();
```

Comment:
why is this removed?

Reply:
In this create_array API, the crop related arrays are created and updated with the crop values, but when it is invoked by the create_node API in the nodes the ROI value is still not updated, hence the array is not updated correctly. So this update_array call is not required here